#! /bin/sh /usr/share/dpatch/dpatch-run
##
## svnshell by <dwhedon@debian.org>
##
## DP: Contains r20992 from upstream trunk, which may be in 1.4.1.

@DPATCH@
Index: subversion-1.7.0/tools/examples/svnshell.py
===================================================================
--- subversion-1.7.0.orig/tools/examples/svnshell.py	2011-10-20 20:47:32.000000000 +0200
+++ subversion-1.7.0/tools/examples/svnshell.py	2011-10-20 21:55:53.000000000 +0200
@@ -56,14 +56,8 @@
   def postcmd(self, stop, line):
     self._setup_prompt()
 
-  _errors = ["Huh?",
-             "Whatchoo talkin' 'bout, Willis?",
-             "Say what?",
-             "Nope.  Not gonna do it.",
-             "Ehh...I don't think so, chief."]
-
   def default(self, line):
-    print(self._errors[randint(0, len(self._errors) - 1)])
+    print "Unknown command."
 
   def do_cat(self, arg):
     """dump the contents of a file"""
